.gitpod.yml

```diff
@@ -1,28 +1,39 @@
 image: mortenson/tome
 tasks:
-  - name: Install tome
-    command: |
-      COMPOSER_MEMORY_LIMIT=2048M composer install
-      [ -d content ] && drush tome:install -y
-      gp sync-done install
   - name: Init and login
-    init: |
-      echo "Waiting for install..."
-      gp sync-await install
     command: |
       eval $(gp env -e DRUSH_OPTIONS_URI="0.0.0.0:8888")
       [ ! -d content ] && echo "Looks like you haven't initialized a Tome site yet - make sure to composer require any profiles you want to use (ex: 'composer require drupal-tome/bookish'), then run drush tome:init to get started!"
       [ -d content ] && drush uli
   - name: Runserver
     init: |
-      echo "Waiting for install..."
-      gp sync-await install
+      COMPOSER_MEMORY_LIMIT=2048M composer install
+      [ -d content ] && drush tome:install -y
```

Comment:
So this is how the gitpod file used to work before my last commit, but when it compiles init/command it does it like this:
```
{
# init commands...
} && {
## command commands...
}
```

And for some reason the "command" section never ran.

Reply:
Also this probably has to run before the drush uli above

Reply:
`tasks` is an array of terminals that would run in parallel (the order of items in the array doesn't matter).

`init` tasks would run during "prebuild", always before `command` tasks, so `composer install` and `drush tome:install` run first, and only after, `drush uli` would run.

I don't think I saw `command` tasks not running, I'll have to test how `drupal-tome` was setup before the latest commit.